gettext-rs/src/macros.rs

```diff
@@ -1,50 +1,85 @@
+#[macro_export]
```

Comment:
We can't avoid exporting this macro, right? Can we at least hide it from the docs, to hopefully prevent people from relying on it?

Reply:
> Can we at least hide it from the docs, to hopefully prevent people from relying on it?

Sounds good. I don't think we can avoid exporting it